### PR TITLE
add latitude payment icons

### PR DESCRIPTION
**Pull request by drajk**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

Website and more info:
https://www.latitudefinancial.com.au/credit-cards/#interest-free-plans

Brand guidelines:
https://company-163189.frontify.com/d/K3sviYKKQp6e

### CODE REVIEW

**Comment by drajk** [comment]
@tauthomas01 - Hi, just making sure that you were notified about this pull request.
Would be great to getbyour feedback on this. 

Thanks.

File: app/assets/images/payment_icons/latitude_creditline_au.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...49-.211l-.122-.217a.167.167 0 00-.185-.051 3.338 3.338 0 01-1 .142 1.585 1.585 0 01-1.123-.46 1.4 1.4 0 01-.356-.923h2.951c0-1.023-.646-1.507-1.704-1.507zm-1.241 1.3a1.13 1.13 0 011.22-1.01c.667 0 1.128.3 1.128 1.012zm1.5-3.084a.368.368 0 00.228-.206v.69a.376.376 0 00.568.372l1.93-.846c.313-.137.313-.362 0-.5l-1.93-.846a.376.376 0 00-.568.372v.68a.368.368 0 00-.228-.206l-1.93-.846a.377.377 0 00-.569.372V9.8a.376.376 0 00.569.372zm2.638 2.366l-.209-.538H33.5v.659h.079v-.5l.2.5h.073l.194-.5v.5h.079v-.659h-.1zm-9.365-.509h-.289v3.079h.61v-2.747a.328.328 0 00-.323-.327zm-2.611-.523a.327.327 0 00-.326-.327h-.312v3.6a.327.327 0 00.309.325h2.238v-.326h-1.911zm-1.993-.328h-.6v.85h-.617v.275h.617v2.09c0 .371.285.776 1.094.776a1.73 1.73 0 00.376-.045v-.291a1.291 1.291 0 01-.206.017c-.433 0-.66-.223-.66-.469v-2.072h.866v-.275h-.866zm4.438.5l.431-.189c.069-.031.069-.081 0-.112l-.431-.188a.085.085 0 00-.128.083v.323a.084.084 0 00.126.09z" fill="#fff"/></g><path d="M0 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3H3a2.946 2.946 0 00-3 3zm35-2a2.006 2.006 0 012 2v18a2.006 2.006 0 01-2 2H3a2.006 2.006 0 01-2-2V3a2.006 2.006 0 012-2z" fill-rule="evenodd" opacity=".5"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/latitude_creditline_au.svg:1]: I don't see the result of this icon. I'm seeing a white background with grey border.

File: app/assets/images/payment_icons/latitude_creditline_nz.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...sDDWhTYLef8k7f1pTQZqvI+UfghKmwQNfjRpLpXncSjmNXkTARsc+grxlnXqQoOdLR5q0F94ff1sNKvRw+xKlIt8JPbUru1cyXRPdtL600xx3fVxJUgZdQPisZ+BACkewiZqfp3LlDFi+SwSJWoX9mYEZoFyPaIhFKrZmOXsp7Pe1JtJ4+zPm6hDmmeAPFZokOVXBi0zNnkeM6i6jkY3LHupVTChBLPVWK5EuIkhSfsSpKlVpKzdlaER3F/YWB7i4EgLB8XVK6FKCizVlTn+lYThfTmEgW2aMh/OlnCKw8+/S0sY1rjnE3U6sR6x5TLmeJfhZZNd6WayfZCDcMZhtCy60W6vdWOjKS2cZEbLSL71niqsR8FiklpqSH5JRUdW1x2/2H6B1gR8KqY/nSzfmG255yFo6so3Kkel5RlTFupWSmqGqi+ULqaQ8bIm6wYCWp2MbJLVYzr8uMNI/uZqnAWaghIykyUr8k0FGeQsu3dUOyIZVP4EiduY1xL+e2vyKFp3swNC75U30AWHSsWlmTDUJH3jYeOispP5HdzKncLD6tt4NZdFe+Ry2W1Yg7mx+TPJG2N4aRQb+vl5YVlVOMKVXP2LPo9M5ZMiS8IHsxg8e9B94hARuTRSFClJXhl3/dt00ULcwPK6EHD1vlGrISSzDZ0m6W+fdKSdpMx9CQRzkbknEyKYQZuP5UgBtmuQxzXWpMBM7VZjUzBc6lEsWtqoKoFuFtFC6wj8PhV7EEC9rGQJyL5y6nXWRn3z1Gb+EOO81unDpcRaJvqqizilWT4gCMzD5QjFJG65geRfR/7LCMBI/lwaQ0kxp5+Mj3md9dvEpDwhxJXEofJRycpKpI4gO3h8zONDX2AOIURER8+1Qwbw6cvPw//7e3DsPjodXJPvIeF9Pn897n897n897ne/kLEWG//FzSFNAAAAAASUVORK5CYII="/></g><path d="M0 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3H3a2.946 2.946 0 00-3 3zm35-2a2.006 2.006 0 012 2v18a2.006 2.006 0 01-2 2H3a2.006 2.006 0 01-2-2V3a2.006 2.006 0 012-2z" fill-rule="evenodd" opacity=".1"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/latitude_creditline_nz.svg:1]: The border seems off - we see the background of the main icon "over-extending"

[alt](https://screenshot.click/08-14-xm0kl-rb66h.png)


You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

File: app/assets/images/payment_icons/latitude_gem_au.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...><path d="M0 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3H3a2.946 2.946 0 00-3 3zm35-2a2.006 2.006 0 012 2v18a2.006 2.006 0 01-2 2H3a2.006 2.006 0 01-2-2V3a2.006 2.006 0 012-2z" fill="#002a66" fill-rule="evenodd" opacity=".3"/><path d="M12.525 12.4a.663.663 0 10.662.663.663.663 0 00-.662-.663zM6.937 8.232H5.611v7.577h7.576v-1.326h-6.25zm3.5 1.326a.663.663 0 10-.659-.658.663.663 0 00.663.658zm2.084.758a.663.663 0 10.662.663.663.663 0 00-.658-.663zm-4.163-.758A.663.663 0 107.7 8.9a.662.662 0 00.658.658zm4.167-1.326a.663.663 0 10.662.663.663.663 0 00-.662-.663zm19.914 1.925h-.422l-1.482 2.533-1.5-2.533h-.425v3.6h.49v-2.61l1.3 2.172h.236l1.3-2.188.006 2.626h.494zm-8.345 1.992H25.9v-.437h-1.806v-1.107h2.028v-.448h-2.543v3.6h2.615v-.448h-2.1zm-5.205-1.4a1.514 1.514 0 01.738-.177 1.452 1.452 0 011.046.416l.319-.319a1.6 1.6 0 00-.6-.414 2.223 2.223 0 00-1.776.1 1.748 1.748 0 00-.689.659 1.915 1.915 0 000 1.892 1.768 1.768 0 00.687.658 2.006 2.006 0 00.983.239 2.327 2.327 0 00.741-.118 1.916 1.916 0 00.623-.34v-1.4h-.494v1.174a1.524 1.524 0 01-.844.231 1.5 1.5 0 01-.734-.18 1.327 1.327 0 01-.512-.5 1.457 1.457 0 010-1.421 1.3 1.3 0 01.512-.497z" fill="#fff"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/latitude_gem_au.svg:1]: Same for this one: [alt](https://screenshot.click/08-18-34638-mblol.png)


You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

File: app/assets/images/payment_icons/latitude_go_au.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+Pj4+P77fvBNjRL2W3Jy/AAAAAAElFTkSuQmCC"/><path d="M-.693 4.777h15.657a1.657 1.657 0 011.658 1.657v11.05a1.657 1.657 0 01-1.658 1.657H-.693z" fill="#0046aa"/></g><path d="M0 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3H3a2.946 2.946 0 00-3 3zm35-2a2.006 2.006 0 012 2v18a2.006 2.006 0 01-2 2H3a2.006 2.006 0 01-2-2V3a2.006 2.006 0 012-2z" fill="#002a66" fill-rule="evenodd" opacity=".3"/><g fill="#fff"><circle cx="8.358" cy="8.895" r=".663"/><path d="M10.441 9.558a.663.663 0 10-.663-.658.663.663 0 00.663.658zM13.187 15.809v-1.326h-6.25v-6.25H5.611v7.576zM12.525 9.558a.663.663 0 10-.663-.663.662.662 0 00.663.663z"/><path d="M12.525 13.725a.663.663 0 10-.663-.663.662.662 0 00.663.663zM12.525 11.642a.663.663 0 10-.663-.663.662.662 0 00.663.663z"/><path d="M26.963 9.985a1.974 1.974 0 101.983 1.974 1.929 1.929 0 00-1.983-1.974zm-5.594-.653a2.565 2.565 0 011.968.793l-.454.455a2.026 2.026 0 00-1.485-.6 1.975 1.975 0 10-.007 3.948 2.159 2.159 0 001.2-.331v-.968h-.975v-.7h1.678v2a3.027 3.027 0 01-1.939.653 2.629 2.629 0 11.015-5.254zm5.594 0a2.628 2.628 0 11-2.729 2.627 2.6 2.6 0 012.729-2.627z" fill-rule="evenodd"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/latitude_go_au.svg:1]: Same for this one

[alt](https://screenshot.click/08-20-88ezv-qhp3r.png)

File: app/assets/images/payment_icons/latitude_gem_nz.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...a2.329 2.329 0 00-1.638-.634 2.825 2.825 0 00-1.643.514.952.952 0 00-.831-.412 1.125 1.125 0 00-.785.3 3.789 3.789 0 00-1.139 2.377v.017A5.7 5.7 0 0120.1 12.31a2.591 2.591 0 01-1.838.755 2.373 2.373 0 01-1.49-.454l4.5-4.5-.307-.336a3.814 3.814 0 00-5.387 0 3.807 3.807 0 000 5.384 3.68 3.68 0 002.682 1.121 3.537 3.537 0 002.7-1.122 7.146 7.146 0 001.531-3.417v-.008c.108-.572.391-1.365.646-1.364.05 0 .2.007.186.426V14.1h1.166V9.055a.974.974 0 01.293-.728 1.622 1.622 0 01.95-.291 1.227 1.227 0 01.814.291.828.828 0 01.282.6V14.1h1.174V9.018a.972.972 0 01.291-.692 1.611 1.611 0 01.948-.29 1.223 1.223 0 01.815.291.839.839 0 01.281.63V14.1zm-15.847-3.63a2.48 2.48 0 01.761-1.836 2.518 2.518 0 011.833-.758 2.407 2.407 0 011.278.311l-3.561 3.561a2.4 2.4 0 01-.311-1.278zm14.99 6.551h.857v-.169h-.688v-.337h.592v-.169h-.592v-.338h.688v-.168h-.857zm-5.388-1.013h.4v1.013h.169v-1.013h.4v-.168h-.977zm-2.667 0h.4v1.013h.169v-1.013h.4v-.168h-.97zm1.738 1.013h.169V15.84h-.169zM21.43 15.84l-.53 1.181h.174l.148-.338h.591l.148.338h.178L21.6 15.84zm-.139.675l.221-.507.222.507z" fill="#fff"/><g clip-path="url(#c)"><g clip-path="url(#d)"><path d="M44.471-29.174h38v23.975h-38z" fill="url(#a)"/></g></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/latitude_gem_nz.svg:1]: Same for this one

[alt](https://screenshot.click/08-21-358u6-gbnny.png)

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi @drajk ,

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline.

Let me know if there are any questions.

**Comment by drajk** [comment]
Thanks @tauthomas01, I will fix these issues with border wnd overflow, and leave a message once done.

**Comment by drajk** [comment]
@tauthomas01 - I have updated the logos now. Please review when you get chance. 
Thanks.

File: app/assets/images/payment_icons/latitude_buyers_edge_au.svg [hunk 6]
```diff
@@ -0,0 +1 @@
...nW3TKzjleIIwv/9VnHuK0zMRCAfbEfcZFWh9DfiM2DO2vHGi2v/khzB1jIxpsgRqAiH4a7VFKj11mXHRFOwgs/i6fKB5/i/9aSjjyiXQz0F4lvgb6HvMQVaMtreTx+LAZe5XeLnQN8GzPYpysu383QD4LHKdy7c3QX/2M53kpcqPAT/3kEPua8ty2u4R73wYltCWrfYFjJpyH1Y15t3XIYPMbuAMtOnYaIDz73577FiSZd2VJvg1YezSLP1vhuujcYQB6rpc5wJ7nOGsbzOHs/WVwF73gvPof73xZy8AX9aVhfy4TvkWXJG5Qb/UKd8wHirZPADKccCRng4f3sMHHegTH74peIvUK+213kq0nVPyfLy+xo9fMR2v5qzUmn8U1KFl0+kSwMdLB0N5AuPfhoa0uzzH2QSzhz9rb+2rMehgtwHLKDl9JKj7gDJKw+VF548YHd131bQYJihyM9qjMXqszXCrjUa+We8MDDQeLTaXdS+OCXZY0NsuRdfOD6jrQneLTUxk4B+Bi1KifuXQbgBAiAoH/mtm0tbJAMQsmqlSI15AqM5PlF9nDDhrl4Z+kkIl94sxWnkwFMopoqgFfdmLaZM4RDruKpEzCAfdpy4Z/1yZsT6wqCEyacutJPBwdHg78FyaILN1wk4Z86AdwIwsBwrnGLIOKplq/fBrpf8LFiABiZps8Z50jpuIEkQK1t5YzozoLBH+nGh41FutqaczmdwHT7IqTwEVRu9UIscY5btWSd70kg8XrWu1/weqGUU6KPCP/Uq72f1/T4Apjpp5x6AL03PoLS+cYJyAe+4eKn4n2ONFXsJEEC33EaWBk4Kf+KExMvVKm6J02X7PrHQtf86gP02rvCAf5bJ4PvikqGV5a7gQTFC/+a7qA7Kp+LTn74ROQv6gm3Z4RBj8QAAAAASUVORK5CYII="/></g><path d="m35 0h-32a3 3 0 0 0 -3 3v18a3 3 0 0 0 3 3h32a3 3 0 0 0 3-3v-18a3 3 0 0 0 -3-3zm2 20.2a2.8 2.8 0 0 1 -2.8 2.8h-30.4a2.8 2.8 0 0 1 -2.8-2.8v-16.4a2.8 2.8 0 0 1 2.8-2.8h30.4a2.8 2.8 0 0 1 2.8 2.8z" fill="#ab6c3d" fill-rule="evenodd" opacity=".5"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/latitude_buyers_edge_au.svg:1]: When I zoom in at 200%, the logo looks pixelated. Is there a way to make it vectorized?

[alt](https://screenshot.click/11-22-8p9gk-ii4ik.png)

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi @drajk ,

Thank you for the changes.
I've left one more comment.

File: app/assets/images/payment_icons/latitude_buyers_edge_au.svg (hunk 6 above)
Comment (drajk) [reply app/assets/images/payment_icons/latitude_buyers_edge_au.svg]: Hi @tauthomas01, 
I have excluded this single asset from the pull request for now and would create a separate PR for it. 
Thanks,

Comment (drajk) [reply app/assets/images/payment_icons/latitude_buyers_edge_au.svg]: Please review this PR as our merchants are keen to use other payment icons in the pull request on our gateway.

**Comment by drajk** [comment]
Hi @tauthomas01,

Just a heads up that I have replied to your previous comment. 
Thanks :)

File: app/assets/images/payment_icons/latitude_creditline_nz.svg [hunk 7]
```diff
@@ -0,0 +1 @@
...fghKmwQNfjRpLpXncSjmNXkTARsc+grxlnXqQoOdLR5q0F94ff1sNKvRw+xKlIt8JPbUru1cyXRPdtL600xx3fVxJUgZdQPisZ+BACkewiZqfp3LlDFi+SwSJWoX9mYEZoFyPaIhFKrZmOXsp7Pe1JtJ4+zPm6hDmmeAPFZokOVXBi0zNnkeM6i6jkY3LHupVTChBLPVWK5EuIkhSfsSpKlVpKzdlaER3F/YWB7i4EgLB8XVK6FKCizVlTn+lYThfTmEgW2aMh/OlnCKw8+/S0sY1rjnE3U6sR6x5TLmeJfhZZNd6WayfZCDcMZhtCy60W6vdWOjKS2cZEbLSL71niqsR8FiklpqSH5JRUdW1x2/2H6B1gR8KqY/nSzfmG255yFo6so3Kkel5RlTFupWSmqGqi+ULqaQ8bIm6wYCWp2MbJLVYzr8uMNI/uZqnAWaghIykyUr8k0FGeQsu3dUOyIZVP4EiduY1xL+e2vyKFp3swNC75U30AWHSsWlmTDUJH3jYeOispP5HdzKncLD6tt4NZdFe+Ry2W1Yg7mx+TPJG2N4aRQb+vl5YVlVOMKVXP2LPo9M5ZMiS8IHsxg8e9B94hARuTRSFClJXhl3/dt00ULcwPK6EHD1vlGrISSzDZ0m6W+fdKSdpMx9CQRzkbknEyKYQZuP5UgBtmuQxzXWpMBM7VZjUzBc6lEsWtqoKoFuFtFC6wj8PhV7EEC9rGQJyL5y6nXWRn3z1Gb+EOO81unDpcRaJvqqizilWT4gCMzD5QjFJG65geRfR/7LCMBI/lwaQ0kxp5+Mj3md9dvEpDwhxJXEofJRycpKpI4gO3h8zONDX2AOIURER8+1Qwbw6cvPw//7e3DsPjodXJPvIeF9Pn897n897n897ne/kLEWG//FzSFNAAAAAASUVORK5CYII="/></g><path d="m0 3v18a3.009 3.009 0 0 0 3 3h32a2.946 2.946 0 0 0 3-3v-18a3.009 3.009 0 0 0 -3-3h-32a2.946 2.946 0 0 0 -3 3zm35-2a2.006 2.006 0 0 1 2 2v18a2.006 2.006 0 0 1 -2 2h-32a2.006 2.006 0 0 1 -2-2v-18a2.006 2.006 0 0 1 2-2z" fill-rule="evenodd" opacity=".1"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/latitude_creditline_nz.svg:1]: This logo looks pixelated. Can we make it vectorized?
[alt](https://screenshot.click/15-03-x5fds-pw6r8.png)

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi @drajk ,

I have one more comment and the rest looks good.

**Comment by drajk** [comment]
@tauthomas01 - 
Thanks again for the review. I have excluded creditline from this pull request.
I will create a follow up PR once those assets are vectorised. 

This PR is ready for review again. 

- 
@drajk

**Review by tauthomas01 — APPROVED** [review]
It looks good.
Thank you for the changes.

**Comment by drajk** [comment]
@tauthomas01 - I was wondering how releases work for this repository, do we have a weekly release cycle ?
Thanks again for reviewing the PR :) 

-
@drajk

**Pull request merged**